database/db_build.sql

```diff
@@ -49,14 +48,13 @@ CREATE TABLE users_careers(
 CREATE TABLE universities_careers(
   uni_id      INTEGER   REFERENCES  universities (id),
   career_id   INTEGER   REFERENCES  careers (id),
-  grade_bagrut INTEGER,
-  grade_psychometric INTEGER,
-  grade_tawjihi INTEGER
+  grade_bagrut TEXT,
+  grade_psychometric TEXT,
+  grade_tawjihi TEXT
```

Comment:
I'm not sure if I'm missing something but is there are a reason why the grades in users table are `integer` and in universities they are `text`?

Reply:
These are the grades added from the form I changed them to text because I was mashing the keyboard to test it :+1:

They can be changed back though but we need some validation on the input fields I think

Reply:
If there is a reason I'm totally fine with leaving them like this!